docs/migration.md

```diff
@@ -901,54 +898,47 @@ processors:
     source_name: "%{facility}"
     ## Set Source Host to `net.peer.name`
     source_host: "%{net.peer.name}"
+
 exporters:
-  sumologic/syslog:
-    ## clear_logs_timestamp is by default set to True.
-    ## If it's set to true, it works like `Enable Timestamp Parsing`,
-    ## and `Time Zone` is going to be taken from `extensions` section.
-    ## There is no possibility to configure several time zones in one exporter.
-    ## clear_logs_timestamp sets to true also behaves like
-    ## `Timestamp Format` would be set to `Automatically detect the format`
-    ## in terms of Installed Collector configuration.
-    clear_logs_timestamp: true
+  sumologic:
+    # unique URL generated for your HTTP Source, this is the address to send data to
+    # deprecated, please use sumologicextension to manage your endpoints
+    # if sumologicextension is not being used, the endpoint is required
+    endpoint: <HTTP_Source_URL>
+    # Compression encoding format, empty string means no compression, default = gzip
+    compress_encoding: {gzip, deflate, ""}
+    # max HTTP request body size in bytes before compression (if applied),
+    # NOTE: this limit does not apply to data sent in otlp format,
+    #   to limit size of otlp requests, please use the batch processor:
+    #   https://github.com/open-telemetry/opentelemetry-collector/tree/v0.71.0/processor/batchprocessor
+    # default = 1_048_576 (1MB)
+    max_request_body_size: <max_request_body_size>
+
+    # format to use when sending logs to Sumo Logic, default = otlp,
+    # NOTE: only `otlp` is supported when used with sumologicextension
+    log_format: {json, text, otlp}
+
```

Comment:
I would avoid adding too much configuration options for customers, epecially for migration from installed collector, as they should use defaults and shouldn't play with log format, request size, compression or endpoint by themselves

```suggestion
  sumologic:

```